src/core/base/render_base.cpp

```diff
@@ -236,15 +236,15 @@ StdPicture AbstractRender_t::LoadPicture(const std::string &path,
 
     RGBQUAD upperColor;
     FreeImage_GetPixelColor(sourceImage, 0, 0, &upperColor);
-    target.ColorUpper.r = upperColor.rgbRed;
-    target.ColorUpper.g = upperColor.rgbGreen;
-    target.ColorUpper.b = upperColor.rgbBlue;
+    target.ColorUpper.r = upperColor.rgbRed / 255.0f;
+    target.ColorUpper.g = upperColor.rgbGreen / 255.0f;
+    target.ColorUpper.b = upperColor.rgbBlue / 255.0f;
 
     RGBQUAD lowerColor;
     FreeImage_GetPixelColor(sourceImage, 0, static_cast<unsigned int>(h - 1), &lowerColor);
-    target.ColorLower.r = lowerColor.rgbRed;
-    target.ColorLower.b = lowerColor.rgbBlue;
-    target.ColorLower.g = lowerColor.rgbGreen;
+    target.ColorLower.r = lowerColor.rgbRed / 255.0f;
+    target.ColorLower.b = lowerColor.rgbBlue / 255.0f;
+    target.ColorLower.g = lowerColor.rgbGreen / 255.0f;
```

Comment:
As far as I could tell, they were `float` fields, invalidly being treated as if they were `uint8_t`s. I had to change this to get the background fill-in to work correctly. I'd personally prefer that we make a class that treats colors as (MSByte)rgba(LSByte) `uint32_t`s, with `constexpr` constructors that take `float`s or `uint8_t`s, but this can be a future change.